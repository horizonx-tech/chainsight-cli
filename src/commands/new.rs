use clap::Parser;

#[derive(Debug, Parser)]
#[command(name = "new")]
/// Generate ChainSight project by prepared template
pub struct NewOpts {}