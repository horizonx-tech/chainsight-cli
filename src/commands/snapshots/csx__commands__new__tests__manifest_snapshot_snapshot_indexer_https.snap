---
source: src/commands/new.rs
expression: template_snapshot_indexer_https_manifest(&id).to_str_as_yaml().unwrap()
---
# yaml-language-server: $schema=https://raw.githubusercontent.com/horizonx-tech/chainsight-cli/main/resources/schema/snapshot_indexer_https.json
version: v1
metadata:
  label: Sample Snapshot Indexer Https
  type: snapshot_indexer_https
  description: ''
  tags:
  - coingecko
  - DAI
  - USD
datasource:
  url: https://api.coingecko.com/api/v3/simple/price
  headers:
    Content-Type: application/json
  queries:
    ids: dai
    vs_currencies: usd
interval: 3600

